A2FootPrintChart.cs

```diff
@@ -822,22 +822,27 @@ protected override void OnRender(ChartControl chartControl, ChartScale chartScal
 					else
 						deltaColor = BarDeltaDownColor.ToDxBrush(RenderTarget);
 
+					float xBar = chartControl.GetXByBarIndex(ChartBars, chartBarIndex - 1) + (float)(chartControl.BarWidth+4 );
+
 					// HELLO
 					if (totalDelta > 0 && barOpenPrice > barClosePrice)
 					{
 						deltaColor = Brushes.Blue.ToDxBrush(RenderTarget);
-//Draw.Text(this, "pbt" + Time[0].ToString(), "HELLO" , 1, Low[0] - TickSize * 2, ChartControl.Properties.ChartText);
-// Draw.Diamond(this, "pbd" + "HELLO" + "legCount", true, Time[0], High[0] + TickSize * 2, Brushes.Red);
-// Draw.Diamond(this, "MyDiamond" + CurrentBar, true, 0, High[0] + 2 * TickSize, Brushes.Blue);
-						Print("footPrint divergence: Trend changing pending context! 1 at " + Time[0].Hour + ":" + Time[0].Minute + ":" + Time[0].Second + " - " + totalDelta.ToString() + "" + Open[0] + " > " +  Close[0]);
+		                // PlaySound(NinjaTrader.Core.Globals.InstallDir + @"\sounds\alert4.wav");
+						// if (xBar > Count * 0.5)
+						Print("FP divergence: Trend to change pending context! 1 " + xBar.ToString() + "/" + Count.ToString() + " at " + Time[0].Hour + ":" + Time[0].Minute + ":" + Time[0].Second + " - " + totalDelta.ToString() + "" + Open[0] + " > " +  Close[0]);
```

Comment:
I suggest we remove the Prints. During a heavy session, prints will impact the performance and hard to watch the output screen as you look at the candlesticks. Good debugging tool, but not so helpful realtime.